greedy_approach/JobSequencing.c

```diff
@@ -0,0 +1,79 @@
+#include<stdio.h>
+
+struct Jobs
+{
+    float profit;
+    int deadline;
+    int id;
+};
+
+void jobSequencing(int n, struct Jobs job_details[100])
```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.